apisix/core/config_etcd.lua

```diff
@@ -453,7 +476,7 @@ function _M.new(key, opts)
         values = nil,
         need_reload = true,
```

Comment:
Hi @nic-chen , currently after removing init_etcd, test would fail in etcd_sync.t that it could not auto update prev_index as shown [here](https://github.com/apache/apisix/pull/2036/checks?check_run_id=1066582120#step:6:107). Since removing init_etcd would not set need_reload to false so it could not get to watch anymore.

Do you have any suggestions on how to modify etcd_sync test file

Reply:
One option is to set `need_reload` to false when init `/consumer`, but I'm not sure changing it would still stay the same with real situation,

Reply:
The problem is not here. Closed.